src/components/Blog/Blog.css

```diff
@@ -95,6 +95,17 @@ table{
 	white-space: nowrap;
 	
 }
+
+.hideTopButton {
+    display: none;
+    left: -100%;
+    transition: all 1s;
```

Comment:
Transition not working in the demo, make it work or remove it remove transition effect